warehouse/models/intermediate/gtfs/gtfs_quality/int_gtfs_quality__no_rt_critical_validation_errors.sql

```diff
@@ -0,0 +1,54 @@
+WITH feed_guideline_index AS (
+    SELECT * FROM {{ ref('int_gtfs_quality__rt_feed_guideline_index') }}
+),
+
+fct_daily_rt_feed_files AS (
+    SELECT * FROM {{ ref('fct_daily_rt_feed_files') }}
+),
+
+fct_daily_rt_feed_validation_notices AS (
+    SELECT * FROM {{ ref('fct_daily_rt_feed_validation_notices') }}
+),
+
+count_files AS (
+    SELECT
+        base64_url,
+        date,
+        SUM(parse_success_file_count) AS rt_files
+    FROM fct_daily_rt_feed_files
+    GROUP BY 1, 2
+),
+
+critical_notices AS (
+    SELECT
+        date,
+        base64_url,
+        SUM(total_notices) AS errors,
+    FROM fct_daily_rt_feed_validation_notices
+    WHERE is_critical
+    GROUP BY 1, 2
+),
+
+int_gtfs_quality__no_rt_critical_validation_errors AS (
+    SELECT
+        idx.date,
+        idx.base64_url,
+        {{ no_rt_critical_validation_errors() }} AS check,
+        {{ compliance() }} AS feature,
+        rt_files,
+        errors,
+        CASE
+            WHEN COALESCE(errors, 0) = 0 THEN "PASS"
+            WHEN rt_files IS null THEN "FAIL"
```

Comment:
Ok, still would want to confirm with him -- not sure if we want to carry that behavior forward.

Reply:
What she said! When we don't have validator data for a given day let's leave null.
A potential addition would be to create a "reason" column as well, which could add some kind of detail. It would be smart for me to create this now...